src/lib/spinel/spinel.h

```diff
@@ -4986,6 +4986,30 @@ enum
 
     SPINEL_PROP_BACKBONE_ROUTER__END = 0x990,
 
+    SPINEL_PROP_BORDER_ROUTER__BEGIN = 0x990,
+
+    /// Enables or Disables DHCPv6 Prefix Delegation
+    /** Format: `b` - Write-Only
```

Comment:
```suggestion
    /** 
      * Format: `b` - Write-Only
```
Let's keep consistent style.